src/Strategy/Scala/Common.hs

```diff
@@ -31,3 +34,19 @@ removeLogPrefixes content = Text.unlines . map removePrefix $ Text.lines content
 
     maybeRemovePrefix :: Text -> Text -> Text
     maybeRemovePrefix prefix candidate = fromMaybe candidate $ Text.stripPrefix (prefix <> " ") candidate
+
+-- | Removes scala standard library from sbt
+-- Refer: https://github.com/sbt/sbt/blob/master/main/src/main/scala/sbt/internal/graph/GraphTransformations.scala#L39
+--
+-- Note that, as general case FOSSA CLI prefers not to report standard library,
+-- to avoid dependency noise, issue noise arising from inclusion of standard library.
+withoutStdLibs :: Graphing Dependency -> Graphing Dependency
```

Comment:
From latest retro, it seems we want to include stdlib for scala. 

todo:
- [ ] include stdlib in final analysis